### ISSUE TITLE
My Status

### ISSUE THREAD

**Issue by masai214** [StephanieR523/travel#1]
An academic hold has been put on my account at class and I've been temporarily dropped from the program. I'm going to continue to work on the project with you guys if that's okay. I don't want to let you down due to my personal stuff. If you guys want to message me or meet outside of class my number is (732)-491-9627

**Comment by StephanieR523** [comment]
of course, If there's anything I can do to help you or get you any info/ resources from the course just let me know what you need. stephanieromanowski@gmail.com 9086706536
